crates/api_common/src/person.rs

```diff
@@ -55,7 +56,7 @@ pub struct SaveUserSettings {
   pub avatar: Option<String>,
   pub banner: Option<String>,
   pub display_name: Option<String>,
-  pub email: Option<String>,
+  pub email: Option<Sensitive>,
```

Comment:
Oops, I missed these changing of core data types, not a fan of this. Isn't it possible to add annotations for tracing to not log these instead?

Or if that's not possible, use a generic `Sensitive<Type>`, so that we can see the type?

Reply:
tracing relies on the debug implementation for how it prints the types, so if the types debug sensitive info then tracing will log sensitive info

Reply:
but yes I can make Sensitive generic over the type it wraps

Reply:
Specifically the issue is that these are members of structs with `#[derive(Debug)]`. I could manually implement debug for each of these structs but that would be painful